### PR TITLE
Implemented travel modes for directions

### PR DESCRIPTION
**Pull request by pobudzey**
User story #10 is pretty much complete
The only thing missing is making clear which travel mode button is currently selected
I'll submit a separate PR once I figure that out because I'd rather merge this code as soon as possible to avoid merging everything at the same time with everyone else

To make sure the feature is working correctly, I'd suggest selecting two locations that are relatively close to each other and for which you know for a fact that walking and driving directions would be different.

travel_modes

### CODE REVIEW

File: app/src/main/java/com/example/campusguide/directions/ChooseDirectionOptions.kt [hunk 1]
```diff
@@ -90,8 +91,13 @@ class ChooseDirectionOptions constructor(private val route: Route?) : DialogFrag
...
+                            // Start the DirectionsActivity activity
+                            val intent = Intent(this.context, DirectionsActivity::class.java).apply {
+                                putExtra("Origin", start)
+                                putExtra("Destination", end)
+                            }
+                            startActivity(intent)
+                            this.dismiss()
                         },
                         DisplayMessageErrorListener(this.requireActivity()))
                 )
```
Comment (FGRCL) [inline app/src/main/java/com/example/campusguide/directions/ChooseDirectionOptions.kt:103]: I'll take care of that, but I think we'll be able to remove that dialog altogether

File: app/src/main/java/com/example/campusguide/map/GoogleMapInitializer.kt [hunk 2]
```diff
@@ -24,13 +25,17 @@ class GoogleMapInitializer constructor(private val activity: FragmentActivity, p
             map.uiSettings.isMyLocationButtonEnabled = false
 
             // Center the map on Hall building
-            val hall = LatLng(45.497290, -73.578824)
-            map.animateCamera(CameraUpdateFactory.newLatLngZoom(hall,
-                Constants.ZOOM_STREET_LVL
-            ))
+            if(mapId == "maps_activity_map") {
```
Comment (FGRCL) [inline app/src/main/java/com/example/campusguide/map/GoogleMapInitializer.kt:28]: Why do we need that if we only have one map?

**Review by FGRCL — APPROVED** [review]

File: app/src/main/java/com/example/campusguide/map/GoogleMapInitializer.kt (hunk 2 above)
Comment (pobudzey) [reply app/src/main/java/com/example/campusguide/map/GoogleMapInitializer.kt:28]: We now have two activities with two different XML layouts and which require two different map fragments

File: app/src/main/java/com/example/campusguide/directions/ChooseDirectionOptions.kt (hunk 1 above)
Comment (pobudzey) [reply app/src/main/java/com/example/campusguide/directions/ChooseDirectionOptions.kt:103]: Sounds good

**Pull request merged**